README.md

```diff
@@ -58,6 +58,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [OpenQL](https://github.com/QE-Lab/OpenQL) - Compiler framework with algorithm libraries, optimizer, scheduler, QEC, mapping, micro-code generator.
 - [PennyLane](https://pennylane.ai) - Cross-platform Python library for differentiable programming of quantum computers.
 - [Perceval](https://github.com/Quandela/Perceval) - [Quandela](https://www.quandela.com)'s software library for programming realistic photonic quantum computers.
+- [Piquasso](https://github.com/Budapest-Quantum-Computing-Group/piquasso) - A photonic quantum computing simulator library written in Python/C++.
```

Comment:
Is this in the correct section? If it's a simulator, it belongs in the simulator section just below. If it's a full stack library, the description should reflect that.

Reply:
Sorry, I moved it to the simulator section.